easybuild/easyconfigs/h/hunspell/hunspell-1.7.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,38 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'hunspell'
+version = '1.7.0'
```

Comment:
@sassy-crick v1.7.1 was released recently, let's bump to that?

Reply:
Done. There was no particular reason for not using the latest version.